.github/workflows/create-release.yml

```diff
@@ -0,0 +1,517 @@
+name: Create Release
+
+on:
+  schedule:
+  - cron: '*/10 * * * *' # every 10 minutes
```

Comment:
It sure is! Fortunately, in the common case where there are no USNs, only the `Poll` job runs. This is still a  concern as stacks scale, though.

FWIW this cron schedule is  less frequent than [the current USN automation](https://github.com/paketo-buildpacks/stack-usns/blob/9b9eea66971ecb791b5c7d9d2b8c6f8db9267ba6/.github/workflows/get-usns.yml#L5). 

I worry about running this job too _infrequently_ since our current source of USNs truth only holds the 10 most recent USNs.

Reply:
Yeah, we actually decreased the frequency from every 5 minutes to every 10 minutes. We might be able to get away with polling less frequently if we can be confident that the USN feed we receive still contains the last seen USNs. See the comment below about USN feed architecture.

If we stick with this architecture, the polling frequency should be about equal to number of USNs per feed (i.e. 10) / maximum rate of USN generation. Obviously the rate of USN generation isn't fixed, so this will always be a statistical approach.

Reply:
For now, let's stick with the cron schedule? We can come back and adjust this if it causes problems.